README.md

```diff
@@ -670,37 +670,22 @@ print(type(post.content))
 class Post(BaseModel):
    content: Video | Article
 ```
-2. Validate input has only valid fields 
+2. Validate input has only valid fields : using [`Extra` Model Configuration](https://pydantic-docs.helpmanual.io/usage/model_config/), choose any of :
+- allow (allow any fields apart from the ones listed)
+- ignore (filter out any fields apart from the ones listed)
+- forbid (raise errors if any fields are present apart from the ones listed)
 ```python
-from pydantic import BaseModel, root_validator
+from pydantic import BaseModel, Extra
 
-class Article(BaseModel):
+class Article(BaseModel, extra=Extra.forbid):
```

Comment:
I think it would be easier to understand the nature of this config if we explicitly set it in the Config class instead of setting it through model class kwarg since it's less popular.

Reply:
Not exactly sure on what you really mean, but have added what I followed from the official docs. Perhaps a code sample might help?

Reply:
sure
```python3
class A(BaseModel):
    class Config:
        extra = Extra.forbid
```

Reply:
Hey! IMO you could also update [#18](https://github.com/zhanymkanov/fastapi-best-practices#18-be-careful-with-dynamic-pydantic-fields) with extra config like a way to prevent unexpected behavior with not-very-specific models